support/networking/client.go

```diff
@@ -0,0 +1,35 @@
+package networking
+
```

Comment:
What is the idea behind using this new class?

Reply:
just  so we have a single client and can pass this as a dependency into other functions/structs that need an http client we now just need to test it once. Also we are separating the more low level calls from say the price feed and following single responsibility